docs/guides/admin/eveapi_admin_contact.md

```diff
@@ -1,22 +1,24 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
-## what
+# Eve API Administrative Contact
+
+## What
 Requests to the EVE API need to have an administrative contact email set before SeAT will queue jobs to process. CCP made the request in [this](https://github.com/eveseat/seat/issues/77) Github issue. To address this, the email address is added to the User-Agent string that is used when making EVE API requests as can be seen [here](https://github.com/eveseat/eveapi/blob/master/src/Helpers/PhealSetup.php#L77).
```

Comment:
Not EVE API anymore
I am also not sure if that is still a requirement? Since you register your SSO-Application with Developers Application Portal